Source/Hydrate.spoon/init.lua

```diff
@@ -0,0 +1,124 @@
+--- Friendly Reminder to remind you to hydrate.
```

Comment:
The first line of the `---` docstrings should name the module:

```lua
--- === Hydrate ===
```

Reply:
thanks